nixos/tests/domination.nix

```diff
@@ -2,6 +2,7 @@ import ./make-test-python.nix ({ pkgs, ... }: {
   name = "domination";
   meta = with pkgs.lib.maintainers; {
     maintainers = [ fgaz ];
+    broken = true; # times out after a couple hours
```

Comment:
Huh. I wonder what changed. The test is still the same but now it doesn't get recognized.

Could you instead change the `wait_for_text` below to

```python
machine.wait_for_text(r"(New Game|Start Server|Load Game|Help Manual|Join Game|About|Play Online)")
```

This fixes it, and should make it much more robust